constraints.txt

```diff
@@ -11,6 +11,7 @@ altair==4.1.0
 anyio==2.2.0
 apache-beam==2.27.0
 appdirs==1.4.4
+appnope==0.1.2
```

Comment:
Please verify this has an appropriate license.

Reply:
Good catch. It is BSD 2-clause. https://github.com/minrk/appnope/blob/main/LICENSE